sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ece-response/_tests_/ece-response-files/CONF/CONR_008.TXT

```diff
@@ -1,4 +1,4 @@
 1AJAACONFIRMATION        20230418                                                                                                                     
 2AJBHDISBNUMBERBCAG000153000999999999LASTNAME                 FIRSTNAME      20010326APPLNUMBER0489311     060202301092023062220230401YENRLDATEN000000
-2AJBH1119353191BCAG000139000999999999LASTNAME                 FIRSTNAME      1995050320225295820497985     060202301092023062220230401Y20230418N000000
+2AJBH1119353191BCAG000139000999999999LASTNAME                 FIRST����      1995050320225295820497985     060202301092023062220230401Y20230418N000000
```

Comment:
Minor typo.

Reply:
THis is added to include ASCII special characters and it works.

Reply:
Thanks. Got it 👍